src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.jsx

```diff
@@ -220,7 +220,11 @@ const SendFunds = ({
                 )}
                 <Row margin="sm">
                   <Col>
-                    <TokenSelectField tokens={tokens} initialValue={selectedToken} />
+                    <TokenSelectField
+                      initialValue={selectedToken}
+                      isValid={tokenAddress && String(tokenAddress.toUpperCase()) !== 'ETHER'}
```

Comment:
shouldn't this be `String(tokenAddress).toUpperCase()`? so the `toUpperCase` is called after you transform the value to a string

Reply:
fixed.

BTW, I couldn't find a scenario where we need to convert to String a `tokenAddress`. But maybe it is an edge case I'm not considering.